ruby-on-rails/ruby-on-rails-quiz.md

```diff
@@ -829,7 +830,7 @@ end
 - [ ] `<%= render partial: "nav", globals: {selected: "about"} %>`
 - [ ] `<%= render partial: "nav", local_variables: {selected: "about"} %>`
 - [x] `<%= render partial: "nav", locals: {selected: "about"} %>`
-- [ ] `<%= render partial: "nav", selected: "about"} %>`
+- [x] `<%= render partial: "nav", selected: "about"} %>`
```

Comment:
This is pretty straightforward. You don't need locals anymore, in fact, you don't need partial keyword anymore either.